### ISSUE TITLE
Training from scratch/ Training Data/ Traning epochs

### ISSUE THREAD

**Issue by WinterStraw** [RVC-Boss/GPT-SoVITS#176]
Thanks a lot for the sharing projects.

When I tried to find out more, I had some questions:
1. Can you share the data information you used to complete the pre-training?
2. What are the recommended training epochs or steps in pretraining and finetune tasks?
I noticed that some of the issues mentioned that overtraining GPT can cause missing text when generating. I'm worried about this.

**Comment by zhuangzhuangliu2345** [comment]
mark

**Comment by RVC-Boss** [comment]
1000hours Chinese+700hours English+300hours Japanese
2 stages training from scratch: https://github.com/RVC-Boss/GPT-SoVITS/issues/67 (remember to set sovits_encoder to freeze_false)
The training data for GPT should pay more attention to the alignment between text and speech, especially in terms of pauses and punctuation, as well as filtering stutters and repetitive speech. For SoVITS training data, better audio quality is better.

**Comment by BankNatchapol** [comment]
@RVC-Boss If possible, could you please provide the configuration or script for training from scratch? There are several aspects in the current script that I'm unsure about, such as the best configuration for parameters like the learning rate (currently, lr in the script is locked at 0.002) and determining which is the SoVITS encoder layers that needed to unfreeze.

**Comment by RVC-Boss** [comment]
> @RVC-Boss If possible, could you please provide the configuration or script for training from scratch? There are several aspects in the current script that I'm unsure about, such as the best configuration for parameters like the learning rate (currently, lr in the script is locked at 0.002) and determining which is the SoVITS encoder layers that needed to unfreeze.

https://github.com/RVC-Boss/GPT-SoVITS/wiki/%E4%BB%8E%E9%9B%B6%E8%AE%AD%E7%BB%83(train-from-scratch)
You can read and modify from webui training codes.

**Comment by tzleon** [comment]
Are there any gender-specific requirements for training data?
